lib/components/Toggle/Toggle.treat.ts

```diff
@@ -16,14 +16,10 @@ export const root = style({
 });
 
 const realFieldBase = style({
-  opacity: 0,
   height: hitArea,
   zIndex: 1,
   selectors: {
     ...debugTouchable(),
-    [`&:not(:disabled)`]: {
```

Comment:
Why was this removed?

Reply:
A toggle does not support being disabled. This was a copy and paste from `InlineField` originally i believe. So moved the `cursor: pointer` to box props (same for your next comment)

Reply:
Nice clean up 👍